zoo-css-challenge/style.css

```
@@ -118,22 +140,38 @@ footer {
 }
 
 .giraffe {
-  background-color: #465775;
+  background-color: #483c46 /* #465775*/;
   color: #56e39f;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+  color: #f4743b /*#ef6f6c*/;
+}
+#strong {
+  font-size: 20px;
+}
+.giraffe p {
+  color: #beee62;
+}
+/*.news h2 {
+  text-align: center;
+}*/
+.Leopard h2{
+  color: #e4ce0b;
+  font-size: 40px;
```

```suggestion
  font-size: 2.5rem;
```
Same as my other comment